cmd/project/samples.go

```diff
@@ -85,3 +105,39 @@ func runSamplesCommand(clients *shared.ClientFactory, cmd *cobra.Command, args [
 	// Execute the `create` command with the set flag
 	return createCmd.ExecuteContext(ctx)
 }
+
+// listSampleSelection outputs available samples matching a language flag filter
+func listSampleSelection(ctx context.Context, clients *shared.ClientFactory, sampleRepos []create.GithubRepo) error {
+	filteredRepos := filterRepos(sampleRepos, samplesLanguageFlag)
+	sortedRepos := sortRepos(filteredRepos)
```

Comment:
question: Are you noticing that the repos aren't sorted?

<img width="762" height="570" alt="Image" src="https://github.com/user-attachments/assets/13f2aa09-8d09-4fc3-ac47-98335592bb0a" />